docs/rules/use-waiters-in-module-scope.md

```diff
@@ -0,0 +1,36 @@
+# ensure waiters are created in module scope (use-waiters-in-module-scope)
+
+Please describe the origin of the rule here.
+
+
+## Rule Details
```

Comment:
There's a better template you can follow [here](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/_TEMPLATE_.md).

Reply:
Oops, forgot that.

Reply:
Fixed.